src/app/home/home.component.html

```diff
@@ -71,13 +71,32 @@ <h5>${{getProgress()}} of ${{getGoal()}} donated.</h5>
 
     <!--HOME PAGE CONTENT GOES HERE-->
     <mat-sidenav-content class="main-content-container">
-        <div class="main-content">
-            <div class="payment-feed-container">
+        <ng-container *ngIf ="mainContentOpened">
+            <div class="main-content">
+                <div class="payment-feed-container">
+                </div>
             </div>
-        </div>
+        </ng-container>
+
+        <ng-container *ngIf ="altViewOpened">
+            <div class ="home-view-placeholder" >
+            </div>
+        </ng-container>
 
         <button (click)="toggleSideNavOpened()" class="vertical-toggle-button">
             {{getSidenavButtonText()}}
         </button>
+
     </mat-sidenav-content>
-</mat-sidenav-container>
\ No newline at end of file
+
+
+
+</mat-sidenav-container>
+
+<div class="altViewNavigation">
```

Comment:
put this within the main content container, that way you can use it to toggle between payment feed and altview. what you should be toggling is the rounded rectangle container in the middle of the home view. , not the entire view.